### PR TITLE
Slider: use default styles on WinUI 3

### LINKED ISSUE #5147: Slider - Use default control templates for WinUI3

### PR DESCRIPTION
**Pull request by antonfirsov**
### Description of Change

Remove most of the custom styling and logic. Reimplement logic for `ThumbImageSource`: switch between a custom and the default `Style` for the Slider's `Thumb` depending on the value of `ThumbImageSource`.

#### Before
Slider-Windows-Before-03

#### After
Slider-Windows-After-01

### Issues Fixed

Fixes #5147

### CODE REVIEW

**Comment by mattleibow** [comment]
Look at those modern sliders!!! 😍😍😍😍😍

Is it possible to use an attached property to add the extra property? My greatest fear is that Big Co is going to base some weird and wacky control on our MauiSlider and we are going to have to maintain this control for eternity. People talk about "we can break some things in net7", but that is never going to happen. We can barely convince runtime to add a critical and required feature after they snap

**Comment by antonfirsov** [comment]
@mattleibow to be honest I don't like the `MauiXyz : Xyz` subclassing pattern at all, and also skeptical about the `XyzExtensions` utility classes. Why to expose such a wide API surface with so many leaking implementation details?

> Is it possible to use an attached property to add the extra property? 

Can't we instead move all the internal state into our `SliderHandler`? (This would mean that some of the `SliderExtensions` methods have to go though.)

File: src/Core/src/Platform/Windows/MauiSlider.cs [hunk 1]
```diff
@@ -64,23 +52,13 @@ public WImageSource ThumbImageSource
 			set { SetValue(ThumbImageSourceProperty, value); }
 		}
 
-		internal event EventHandler Ready;
-
 		protected override void OnApplyTemplate()
```
Comment (antonfirsov) [inline src/Core/src/Platform/Windows/MauiSlider.cs:55]: @mattleibow another tricky part about getting rid of `MauiSlider` is that the logic in `OnApplyTemplate` has to move into an event handler. In my experiments using the `Loaded` event seemed to work well for this, but I know little about the actual control lifecycle.

**Comment by antonfirsov** [comment]
Okkay, so for now I'm keeping around the state in `MauiSlider` and making it internal.

**Review by mattleibow — APPROVED** [review]

**Pull request merged**